index.html

```diff
@@ -13,43 +13,132 @@
-                        <div class="col-sm-6 col-md-9">
-                            <div class="card-body">
-                                <div class="input-group mb-1">
-                                    <input type="text" class="form-control" placeholder="Name" aria-label="Name">
-                                </div>
-                                <div class="input-group mb-1">
-                                    <input type="text" class="form-control" placeholder="Mobile Number" aria-label="Mobile Number">
-                                </div>
-                                <div class="input-group mb-1">
-                                    <input type="text" class="form-control" placeholder="Email" aria-label="Email">
-                                </div>
-                                <div class="input-group mb-1">
-                                    <input type="text" class="form-control" placeholder="Address" aria-label="Address">
-                                </div>
+                    </div>
+                    <div class="col-sm-6 col-md-9">
+                        <div class="card-body">
+                            <div class="input-group mb-1">
+                                <input type="text" class="form-control" placeholder="Name" aria-label="Name">
+                            </div>
+                            <div class="input-group mb-1">
+                                <input type="text" class="form-control" placeholder="Mobile Number" aria-label="Mobile Number">
+                            </div>
+                            <div class="input-group mb-1">
+                                <input type="text" class="form-control" placeholder="Email" aria-label="Email">
+                            </div>
+                            <div class="input-group mb-1">
+                                <input type="text" class="form-control" placeholder="Address" aria-label="Address">
                             </div>
                         </div>
                     </div>
-            </section>
-    </div>
+                </div>
+        </section>
+</div>
+
+
+     <div class="container">
+        <div class="personal-details gap-2 d-grid">
+            <h2>Personal Details</h2>
+           
+    <div class="personal-details my-1">
+      <div class="col-lg-8">
+        <div class="card mb-4">
+          <div class="card-body">
+            <div class="row">
+              <div class="col-sm-3">
+                <p class="mb-0">Full Name</p>
+              </div>
+              <div class="col-sm-9">
+                <p class="text-muted mb-0 text-capitalize">Priya Vijay Pal</p>
+              </div>
+            </div>
+            <hr>
+            <div class="row">
+              <div class="col-sm-3">
+                <p class="mb-0">Email</p>
+              </div>
+              <div class="col-sm-9">
+                <p class="text-muted mb-0 text-lowercase">priyapal20009696@gmail.com</p>
```

Comment:
The email link should be open their default email person.

Reply:
This changes from Ajay D.